pkg/catalogv2/content/content.go

```diff
@@ -76,6 +86,15 @@ func (c *Manager) Index(namespace, name string) (*repo.IndexFile, error) {
 		return nil, err
 	}
 
+	c.lock.RLock()
+	if cache, ok := c.IndexCache[fmt.Sprintf("%s/%s", r.status.IndexConfigMapNamespace, r.status.IndexConfigMapName)]; ok {
+		if bytes.Compare(cache.binaryData, cm.BinaryData["content"]) == 0 {
```

Comment:
Why don't you just keep the revision number of the config map. Faster comparison.

Reply:
oh you are right